src/Internal/QueryM/Ogmios.purs

```diff
@@ -331,6 +402,514 @@ instance DecodeAeson MempoolSnapshotAcquired where
     map AwaitAcquired <<< aesonObject
       (flip getField "AwaitAcquired" >=> flip getField "slot")
 
+-- | The acquired snapshot’s size (in bytes), number of transactions, and capacity
+-- | (in bytes).
+newtype MempoolSizeAndCapacity = MempoolSizeAndCapacity
+  { capacity :: BigInt
+  , currentSize :: BigInt
+  , numberOfTxs :: BigInt
```

Comment:
These should be just `Int`s - we are sure they are not going to overflow, right?